### PR TITLE
OPS-5833 Publish Enterprise Edition Docker Notebook image for Aerospike Database 6.0.0.1 with Aerospike Connect for Spark 3.4.0_spark3.1

### PR DESCRIPTION
**Pull request by pvinh-spike**

### CODE REVIEW

**Review by neelp-git — COMMENTED** [review]
The last upgrade to the Spark Connector was a breaking upgrade and therefore had to be retracted. 

@rahul-aerospike - is there a document that describes the changes needed in the setup/app to make the old Spark notebooks work with the new version?

**Review by mcoberly2 — APPROVED** [review]
Approving as the file downloads.

@neelp-git Question: Currently, the "All shaded JAR" is specified
Do we want to use the 
"All shaded JAR"  
OR 
"All (but Java client) shaded JAR"  ?

**Comment by rahul-aerospike** [comment]
@pvinh-spike which part is failing?

**Comment by pvinh-spike** [comment]
@rahul-aerospike, nothing is failing yet, just want to make sure @neelp-git ok with the update before create/publish new image.

**Comment by neelp-git** [comment]
@mcoberly2 re: allshaded vs unshaded jar, I would have to try both in a test environment to see if one works better than the other. Unless there is a recommendation from the Spark Connector team. Before I do that, however, I need to know what changes to make for the upgrade, which I know from the previous version upgrade attempt, is not backward compatible with the current version in the notebook. Hope @rahul-aerospike has the answer.

**Comment by rahul-aerospike** [comment]
@neelp-git Please use `unshaded` version. With `unshaded` version, one may get an instance of client and experiment directly with DB, without spark interface.

**Comment by neelp-git** [comment]
OK, thanks. I will follow up with the notebook setup/changes for the new
Spark Connector version.

File: Dockerfile [hunk 1]
```diff
@@ -25,7 +25,7 @@ RUN mkdir /opt/spark-nb; cd /opt/spark-nb\
   && wget -qO- "https://javadl.oracle.com/webapps/download/AutoDL?BundleId=245467_4d5417147a92418ea8b615e228bb6935" | tar -xvz \
   && wget -qO- "https://archive.apache.org/dist/spark/spark-3.0.3/spark-3.0.3-bin-hadoop3.2.tgz" | tar -xvz \
```
Comment (neelp-git) [inline Dockerfile:26]: Need this changed to:
  && wget -qO- "https://archive.apache.org/dist/spark/spark-3.1.2/spark-3.1.2-bin-hadoop3.2.tgz" | tar -xvz \

File: Dockerfile [hunk 2]
```diff
@@ -25,7 +25,7 @@ RUN mkdir /opt/spark-nb; cd /opt/spark-nb\
...
   && pip install findspark numpy pandas matplotlib sklearn \
-  && wget "https://aerospike.com/artifacts/aerospike-spark/3.2.0/aerospike-spark-assembly-3.2.0.jar"
+  && wget "https://download.aerospike.com/artifacts/aerospike-spark/3.4.0_spark3.1/aerospike-spark-3.4.0_spark_3.1_allshaded.jar"
```
Comment (neelp-git) [inline Dockerfile:28]: Need to use the unshaded jar:

&& wget "https://download.aerospike.com/artifacts/aerospike-spark/3.4.0_spark3.1/aerospike-spark-3.4.0_spark_3.1_clientunshaded.jar"

**Review by neelp-git — APPROVED** [review]

**Comment by neelp-git** [comment]
Merging as the updated Spark notebooks are breaking without the new Dockerfile.

**Pull request merged**